app/app/context.py

```diff
@@ -64,7 +64,7 @@ def insert_settings(request):
     if context['github_handle']:
         context['unclaimed_tips'] = Tip.objects.filter(
             expires_date__gte=timezone.now(), receive_txid='', username__iexact=context['github_handle']
-        )
+        ).exclude(txid='')
```

Comment:
Should we just query for it like this versus reducing the QS results?

Reply:
> like this 

like what?